src/playground/App.js

```diff
@@ -21,6 +21,28 @@ const rulesMeta = Array.from(rules.entries()).reduce((result, [key, value]) => {
     return result;
 }, {});
 
+const fillOptionsDefaults = (options = {}) => ({
+    ...options,
+    env: {
+        es6: true,
+        ...options.env
+    },
+    parserOptions: {
+        ...options.parserOptions,
+        ecmaFeatures: {
+            ecmaVersion: "latest",
+            sourceType: "script",
```

Comment:
`ecmaVersion` and `sourceType` are properties of `parserOptions`, not `ecmaFeatures`.